tf-modules/ingest/cloudwatch-events.tf

```diff
@@ -30,30 +30,6 @@ resource "aws_lambda_permission" "generic_lambda" {
   principal     = "events.amazonaws.com"
 }
 
-# StartSF Watcher
```

Comment:
Probably out of scope here, but could/should we change the background processing queue to use an SQS event source mapping instead of a Cloudwatch rule? I ask only because it's using pretty much the exact same code from `sf-starter.js` for the `sqs2sfThrottle` lambda

Reply:
I thought about but the obvious difference in implementation is that the cloudwatch rule inherently limits the number of concurrent executions, which I believe we've discussed as something that should not be a factor in the backgroundProcessing semaphore logic, but the complications and debugging if it *does* turn out to be a factor would be completely different between `sqs2sf` and `sqs2sfThrottle`.

Reply:
I would like to explore doing it, but not in this PR.

Reply:
Makes sense 👍